### PR TITLE
#2 From inputs functionality

### PR DESCRIPTION
**Pull request by alihammad99**
I have added additional variables, and functions for each form input.

### CODE REVIEW

**Comment by pratapdd** [comment]
your commit message
feat(input):added additional input variables

File: starterOnly/modal.js [hunk 1]
```diff
@@ -27,3 +27,79 @@ function closeWindow() {
...
+  country = document.getElementsByClassName("location"),
+  agreeToTerms = document.getElementById("checkbox1"),
+  form = document.getElementById("form"),
+  submit = document.getElementById("submit"),
+  errorMsg = document.getElementsByClassName("error");
+
+  // ---------------- Form Validation ----------------
+
+// ---- First Name Validation
+const checkFirstName = (id, serial, message) => {
```
Comment (pratapdd) [inline starterOnly/modal.js:45]: what are arrow functions. what are it's advantages

File: starterOnly/modal.js [hunk 2]
```diff
@@ -27,3 +27,79 @@ function closeWindow() {
   }
 }
 
+// Form Variables
+const firstName = document.getElementById("first"),
```
Comment (pratapdd) [inline starterOnly/modal.js:31]: what is variable hoisting?

var, let , and const

File: starterOnly/modal.js [hunk 3]
```diff
@@ -27,3 +27,79 @@ function closeWindow() {
...
+  form = document.getElementById("form"),
+  submit = document.getElementById("submit"),
+  errorMsg = document.getElementsByClassName("error");
+
+  // ---------------- Form Validation ----------------
+
+// ---- First Name Validation
+const checkFirstName = (id, serial, message) => {
+  errorMsg[serial].innerHTML =
+    id.value === "" || id.value.length < 2 ? message : "";
```
Comment (pratapdd) [inline starterOnly/modal.js:47]: (id.value && id.value.length < 2 ) ? message : ""

File: starterOnly/modal.js [hunk 4]
```diff
@@ -27,3 +27,79 @@ function closeWindow() {
...
+  verification.FirstName =
+    id.value === "" || id.value.length < 2 ? false : true;
+};
+
+// ---- Last Name Validation
+const checkLastName = (id, serial, message) => {
+  errorMsg[serial].innerHTML =
+    id.value === "" || id.value.length < 2 ? message : "";
```
Comment (pratapdd) [inline starterOnly/modal.js:55]: good conventions for readable - 
idValues => array
idValue => string / array

File: starterOnly/modal.js [hunk 5]
```diff
@@ -27,3 +27,79 @@ function closeWindow() {
...
+  verification.LastName = id.value === "" || id.value.length < 2 ? false : true;
```
Comment (pratapdd) [inline starterOnly/modal.js:56]: revisit this condition?

Comment (pratapdd) [reply starterOnly/modal.js:56]: verification.LastName =  !(id.value === "" || id.value.length < 2)

File: starterOnly/modal.js [hunk 6]
```diff
@@ -27,3 +27,79 @@ function closeWindow() {
...
+
+// ---- Quantity Validation
+const checkQuantity = (id, serial, message) => {
+  errorMsg[serial].innerHTML = id.value === "" ? message : "";
+  verification.Quantity = id.value === "" ? false : true;
+};
+
+// ---- Location Validation
+const checkLocation = (serial, message) => {
+  for (let i = 0; i < 6; i++) {
```
Comment (pratapdd) [inline starterOnly/modal.js:81]: never use this - 
i , j 

for (let index = 0; index < 6; index++) {

}

File: starterOnly/modal.js [hunk 7]
```diff
@@ -27,3 +27,79 @@ function closeWindow() {
...
+      return;
+    }
+    verification.Location = false;
+    errorMsg[serial].innerHTML = message;
+    document.getElementsByClassName("checkbox-icon")[i].style.borderColor =
+      "red";
+  }
+};
+
+// ---- Terms Agreement Validation
```
Comment (pratapdd) [inline starterOnly/modal.js:101]: try to use jsdocs convention for comments

https://jsdoc.app/howto-es2015-classes.html

Comment (alihammad99) [reply starterOnly/modal.js:101]: Done

File: starterOnly/modal.js (hunk 6 above)
Comment (alihammad99) [reply starterOnly/modal.js:81]: Updated

File: starterOnly/modal.js (hunk 5 above)
Comment (alihammad99) [reply starterOnly/modal.js:56]: Done

File: starterOnly/modal.js (hunk 3 above)
Comment (alihammad99) [reply starterOnly/modal.js:47]: Edited

**Pull request merged**